libraries/AP_RangeFinder/AP_RangeFinder_VL53L0X.cpp

```diff
@@ -775,7 +775,7 @@ void AP_RangeFinder_VL53L0X::update(void)
 void AP_RangeFinder_VL53L0X::timer(void)
 {
     uint16_t range_mm;
-    if (get_reading(range_mm) && range_mm < 8000) {
+    if (get_reading(range_mm) && range_mm <= 2000) {
```

Comment:
That doesn't seem right, I don't know why the value of 8m had been chosen but 2m is definitely not enough to cover the range of the VL53L1X which has a maximum range of 4m